server/app/assets/javascripts/file_upload.ts

```diff
@@ -26,12 +28,25 @@ function validateFileUploadQuestions(formEl: Element): boolean {
     )
     const isValid = fileInput.value != ''
 
-    // Toggle the error div if invalid.
     const errorDiv = question.querySelector('.cf-fileupload-error')
     if (errorDiv) {
-      // TODO(#1878): Update button aria attributes on error.
+      // Toggle the error div if invalid.
       errorDiv.classList.toggle('hidden', isValid)
     }
+    if (errorDiv && !isValid && !wasSetInvalid) {
+      // Add extra aria attributes to input if there is an error.
+      const errorId = errorDiv.getAttribute('id')
+      if (errorId) {
+        // Only allow this to be done once so we don't repeatedly append the error id.
+        wasSetInvalid = true
+        fileInput.setAttribute('aria-invalid', 'true')
+        const ariaDescribedBy = fileInput.getAttribute('aria-describedBy')
+        fileInput.setAttribute(
+          'aria-describedBy',
+          errorId + ' ' + ariaDescribedBy,
```

Comment:
Not blocking: This can be written as an interpolated string.

```
fileInput.setAttribute(
  'aria-describedBy',
  `${errorId} ${ariaDescribedBy}`,
)
```

Reply:
Nice, thanks!